tests/foreman/api/test_setting.py

```diff
@@ -31,23 +32,35 @@ def test_positive_update_login_page_footer_text(self):
         :id: 91c5373d-b928-419d-8509-761adf5b94b0
 
         :expectedresults: Parameter is updated successfully
-
-        :caseautomation: notautomated
         """
+        login_text_id = [ele.id for ele in entities.Setting().search(query={
+            "per_page": 200,
+            'search': 'name="login_text"'
+        })][0]
+        login = entities.Setting(id=login_text_id).read()
+        for login_text in valid_data_list():
+            with self.subTest(login_text):
+                login.value = login_text
+                login = login.update(['value'])
+                self.assertEqual(login.value, login_text)
 
-    @stubbed()
     @tier1
     def test_positive_update_login_page_footer_text_without_value(self):
         """Updates parameter "login_text" without any string (empty value)
 
         :id: 7a56f194-8bde-4dbf-9993-62eb6ab10733
 
         :expectedresults: login_text has empty value after update
-
-        :caseautomation: notautomated
         """
+        login_text_id = [ele.id for ele in entities.Setting().search(query={
+            "per_page": 200,
+            'search': 'name="login_text"'
+        })][0]
+        login = entities.Setting(id=login_text_id).read()
+        login.value = ""
+        login = login.update(['value'])
+        self.assertEqual(login.value, "")
```

Comment:
Is there some default value for the footer text? Maybe you should revert the setting back to default at the end of the test so that you don't infuence your other current/future tests. Also I wonder if those tests could be run in multiple threads, if there's a possibility o race condition, pls add  @run_in_one_thread()

Reply:
As this is the message text field which is shown on login page and no other entity is dependant on this parameter, so  it won't influence any other current/future tests and yes, I agree with you for @run_in_one_thread(), there may be chances, will update.